src/components/log/ProfileLogView.js

```diff
@@ -19,7 +29,9 @@ class ProfileLogView extends PureComponent {
             .filter(markerIndex => {
               const data = markers.data[markerIndex];
               return (
-                data !== null && 'category' in data && data.category === 'log'
+                data !== null &&
+                data.type === 'tracing' &&
```

Comment:
I don't have any saved profiles with logs, and the logging features has been removed from the profiler platform in the meantime. So I think we should just remove the Log tab completely.
I should have filed a bug about that when the Gecko change landed. Apologies.